apps/wallet-mobile/src/features/Discover/useCases/ReviewTransaction/ReviewTransaction.tsx

```diff
@@ -63,8 +65,13 @@ export const ReviewTransaction = () => {
       return
     }
 
-    const signature = await signTxWithHW(params.cbor, params.partial)
-    params.onConfirm(signature)
+    signTxWithHW(
```

Comment:
@banklesss I'm guessing u r moving it to the new funnel, this will break ur changes